sensum-pwa/src/state/sensations/State_Sensations.res

```diff
@@ -19,3 +19,15 @@ let sensationsIndex = Recoil.asyncSelector({
     }
   },
 })
+
+// TODO: Instead of a list we could use a more refined type that encapsulates more things
+let sensationsLoaded: Recoil.readWrite<list<Types.sensation>> = Recoil.atom({
+  key: "sensationsLoaded",
+  default: list{},
+})
+
+let useSensations = () => {
+  let (sensations, setSensations) = Recoil.useRecoilState(sensationsLoaded)
+  // TODO: Provide custom setSensations method that actually performs the fetch!
```

Comment:
maybe something like: fetchFromTo: (Types.Bigint.t, Types.BigInt.t) => { /* do fetchings and update loadad sensations*/ }